### DISCUSSION TITLE
[Feature] Apply Sidecar Metadata to Image

### DISCUSSION THREAD

**Discussion by tornupnegatives** [immich-app/immich#14111] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I need a solution to apply metadata during export, but not export metadata, from Immich.

I take great care to apply datestamps to my scans when I shoot 35mm film. However, the sidecar metadata doesn’t get applied when I export the images from Immich.

I’d love to be able to toggle the ability sidecar metadata to the images during export.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10592

**Reply by tornupnegatives** [reply]
I'm sorry, but I don't believe this is a duplicate. I am not asking for the ability to _download_ sidecar metadata. I am asking for ability to _apply_ sidecar metadata during download. I am going to re-work the feature request body to be more clear.
